### DISCUSSION TITLE
上传图片生产的链接与图片管理中图片右键打开的链接地址为何不同？哪个链接是正确的链接？

### DISCUSSION THREAD

**Discussion by heiyi110** [lsky-org/lsky-pro#599] [问答]
“上传图片”生成的链接 与 “图片管理” 中图片右键打开的链接地址为何不同？那个链接是正确的链接？
图片是是同一张图片
“上传图片”生成的链接为XXXX/images/2023/01/21/63cabb18db6c3.png
 “图片管理” 中图片右键打开的链接为XXXX/thumbnails/9a8b598c26c15dfa4d50ffd70d932478.png

**Comment by RobinLDW** [comment]
这个问题有答案了么？ lskypro中已经上传的图片，找不到MD地址呢？？？

**Reply by heiyi110** [reply]
找不到markdown链接的意思吗？在我的图片里右键

**Comment by musnows** [comment]
已经上传的图片，在`我的图片`里面直接右键复制图像连接，就是正确的图片地址。

在`图片管理`里面，需要**打开大图**，再右键复制图片链接。如果不打开大图，复制的是缩略图`thumbnails`
